src/beacons/permissions.ts

```diff
@@ -115,7 +115,17 @@ export const checkPermissionStatuses = async () => {
 export const allowedPermissionsForBeacons = async () => {
   const permissionStatuses = await checkPermissionStatuses();
   const kettleModulesArray = [];
-  if (permissionStatuses.bluetooth) {
+  if (Platform.OS === 'ios') {
+    const majorVersionIOS = parseInt(Platform.Version, 10);
+    // For iOS 13 and above, Bluetooth permission is not required
+    // Then it should send bluetooth as on in the kettle module
+    // NOTE: This is a fix for iOS 12 and above were the SDK crashes if bluetooth is not on
+    if (majorVersionIOS < 13) {
+      kettleModulesArray.push(KettleModules.BLUETOOTH);
+    } else if (permissionStatuses.bluetooth) {
+      kettleModulesArray.push(KettleModules.BLUETOOTH);
+    }
+  } else if (permissionStatuses.bluetooth) {
```

Comment:
To make this logic more clear I can suggest something like
```suggestion ts
  const majorPlatformVersion = parseInt(String(Platform.Version), 10);
  const bluetoothPermissionNotNeeded =
    Platform.OS === 'ios' && majorPlatformVersion < 13;

  if (permissionStatuses.bluetooth || bluetoothPermissionNotNeeded) {
```